From 2e2f1546d77514b4cb0c65c7637b4068ab9e232b Mon Sep 17 00:00:00 2001
From: MikeBeaton <mjsbeaton@gmail.com>
Date: Mon, 23 May 2022 17:04:22 +0100
Subject: [PATCH] OcMainLib DEBUG ONLY

---
 Library/OcMainLib/OpenCoreUefi.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Library/OcMainLib/OpenCoreUefi.c b/Library/OcMainLib/OpenCoreUefi.c
index 8ba1a214..f3ced481 100644
--- a/Library/OcMainLib/OpenCoreUefi.c
+++ b/Library/OcMainLib/OpenCoreUefi.c
@@ -970,6 +970,9 @@ OcLoadUefiSupport (
 
     OcConnectDrivers ();
     DEBUG ((DEBUG_INFO, "OC: Connecting drivers done...\n"));
+#if defined(OC_TARGET_NOOPT)
+    WaitForKeyPress (L"Mike...");
+#endif
   } else {
     OcLoadDrivers (Storage, Config, NULL, FALSE);
   }
-- 
2.31.1

