From d486eefff142523c16837ec9863366afa1643143 Mon Sep 17 00:00:00 2001
From: Mike Beaton <mjsbeaton@gmail.com>
Date: Sun, 30 Oct 2022 07:51:52 +0000
Subject: [PATCH] mike

---
 Library/OcMainLib/OpenCoreUefi.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Library/OcMainLib/OpenCoreUefi.c b/Library/OcMainLib/OpenCoreUefi.c
index 5f0c503e..10a2fca2 100644
--- a/Library/OcMainLib/OpenCoreUefi.c
+++ b/Library/OcMainLib/OpenCoreUefi.c
@@ -254,6 +254,11 @@ OcLoadDrivers (
       FreePool ((CHAR8 *)UnescapedArguments);
     }
 
+#if defined(OC_TARGET_NOOPT)
+    if (Index == 0) {
+      WaitForKeyPress (L"Mike...");
+    }
+#endif
     Status = gBS->StartImage (
                     ImageHandle,
                     NULL,
-- 
2.36.1

